sdk/lusid/utilities/api_client_builder.py

```diff
@@ -152,7 +154,14 @@ def build(cls, api_secrets_filename=None, okta_response_handler=None, api_config
                 )
 
         # Create and return the ApiClient
-        return ApiClient(
+        api_client = ApiClient(
             configuration=config,
             header_name="X-LUSID-Application" if configuration.app_name is not None else None,
             header_value=configuration.app_name)
+
+        # set a correlation id for all requests initiated with this ApiClient
+        corr_id = correlation_id or os.getenv("FBN_CORRELATION_ID")
+        if corr_id is not None:
+            api_client.set_default_header("CorrelationId", corr_id)
```

Comment:
Should we set both these headers in the same way for consistency? It's strange that the __init__ method only allows you to provide a single header.

Reply:
done